processor/src/main/java/org/mapstruct/ap/internal/model/source/MappingOptions.java

```diff
@@ -39,11 +39,16 @@
 
     private static final Pattern JAVA_EXPRESSION = Pattern.compile( "^java\\((.*)\\)$" );
 
+    public String getCondition() {
```

Comment:
Can you please put this getter next to the other getters in this class.

Reply:
yep